### ISSUE TITLE
HTML Renderer - Fetch Database Comments 

### ISSUE THREAD

**Issue by hedgehog-jacek** [Wisser/Jailer#97]
### Is your feature request related to a problem? Please describe.

This feature request is to add comments from the database to the HTML renderer documentation.

![image](https://user-images.githubusercontent.com/95072960/195376247-032d361b-e220-487d-b1fb-fce349118aa6.png)

Database objects can have comments.

E.g., in Postgres a comment can be added to a table like so
```sql
COMMENT ON TABLE hedgehog.us_zip_codes IS 'Store ZIP Codes with the associated city and state.';
```

It would be useful to see these comments displayed within the HTML documentation.

### Describe the solution you'd like

Database comments could be displayed like so in the documentation:

![image](https://user-images.githubusercontent.com/95072960/195375358-60235aff-3c18-4e33-a847-4f1365f2b796.png)

**Comment by Wisser** [comment]
Implemented in release 12.8.1. Please have a look.

**Comment by hedgehog-jacek** [comment]
Thanks for adding this.

I ended up using SchemaSpy for database documentation.

I currently don’t have time to test this feature.
